vue/components/SharedSubComponents/infiniteLoaderMixin.js

```diff
@@ -0,0 +1,90 @@
+const scrollThreshold = 400
```

Comment:
I'm not a fan of the directory name—`SharedSubComponents` is a bit vague as to the _purpose_ of the directory. It's like naming a module "utils". It risks becoming a junk-drawer miscellany. But I don't dislike it enough to block this PR, I just wanted to flag that for your future consideration.